test/lit/passes/optimize-instructions.wast

```diff
@@ -2638,12 +2638,8 @@
   )
   ;; CHECK:      (func $lt_s-sext-zero (param $0 i32) (result i32)
   ;; CHECK-NEXT:  (i32.lt_s
-  ;; CHECK-NEXT:   (i32.shr_s
-  ;; CHECK-NEXT:    (i32.shl
-  ;; CHECK-NEXT:     (local.get $0)
-  ;; CHECK-NEXT:     (i32.const 24)
-  ;; CHECK-NEXT:    )
-  ;; CHECK-NEXT:    (i32.const 24)
+  ;; CHECK-NEXT:   (i32.extend8_s
```

Comment:
Since we already have test/lit/passes/optimize-instructions-sign-ext.wast, how about we pass `--mvp` in this file so we don't lose coverage of the optimizations without sign extension enabled?

Reply:
Great idea!  Done